### PR TITLE
Added documentation about vault key spaces

### PR DESCRIPTION
**Pull request by blarghmatey**
Added a writeup of our discussion about how to manage static secrets
in Vault.

### CODE REVIEW

File: docs/standards/vault.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+- micromasters
+
+### Putting It All Together
+Putting all of this together, a keypath can be templated as:
+
+```
+secret-{{ business_unit }}/{{ environment || global }}/{{ purpose? }}/{{ app_name }}-{{ key_name }}
+```
+
+## Data Conventions
```
Comment (shaidar) [inline docs/standards/vault.md]: This section is a bit unclear to me. When writing a key-value should the full path not be specified? Same thing for the salt state.

Comment (shaidar) [inline docs/standards/vault.md]: Guess what I'm missing is how does the GET request to that URL know what key you are requesting the value for.

Comment (blarghmatey) [inline docs/standards/vault.md]: The GET request will just return all of the data for a given path. For example, if the path `secret/foo/bar` has the data `foo=bar` then when you run the GET request you will get back `foo=bar`. If the data for that path is `foo=bar, baz=quux` then that is what gets returned when you send the GET request.

**Comment by shaidar** [comment]
:thumbsup:

**Pull request merged**